### PR TITLE
Story

### PR DESCRIPTION
**Pull request by grace-p-03**
Contact section approved and merged. I have now checked the HTML in a validator and checked my CSS in the inspector. :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+ <!DOCTYPE html>
+<html lang="en">
+  <head>
+	<title>Grace Palacz | Portfolio</title>
+	<link rel="shortcut icon" type="image/jpg" href="favicon.ico"/>
+	<link href= "normalize.css" type="text/css" rel="stylesheet"/>
+	<link href="style.css" rel="stylesheet" type="text/css" />
```
Comment (charliecog) [inline index.html]: best practice is to move your custom styles to be the last thing (after the script tag)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+header .intro h1 {
+	font-weight: 900;
+	margin: 0 auto;
+	font-size: 5rem;
+}
+
+header .intro {
+	text-align: center;
+	padding-top: 350px;
```
Comment (charliecog) [inline style.css]: dont need the text-align or padding top here, can instead add align-items: center on the header element (the parent)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+			          <a href="https://www.linkedin.com/in/grace-palacz-60134783/" target="_blank">
+					  <i class="fab fa-linkedin-in"></i></a>
+			          <a href="https://github.com/grace-p-03" target="_blank">
+					  <i class="fab fa-github"></i></a>
+			          <a href="https://www.instagram.com/graceisdesigning/" target="_blank">
+			          <i class="fab fa-instagram"></i></a>
+			        </div>	
+			    </div> 
+			      <footer>
+			        <p>© Copyright 2022</p>
```
Comment (charliecog) [inline index.html]: the copyright symbol needs a look (luke sending to slack)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+		
+	}
+	
+}
+
+
+@media only screen and (max-width: 800px) {
+	header {
+		height: 600px;
+		display: flex;
```
Comment (charliecog) [inline style.css]: display flex and justify content arent needed here as they are already included above

File: style.css [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+
+@media only screen and (max-width: 1000px) {
+	
+	nav, nav a {
+		font-size: 1.5rem;	
+	}	
+	
+	header {
+		height: 600px;
+		display: flex;
```
Comment (charliecog) [inline style.css]: display flex and justify content arent needed here as they are already included above

File: style.css [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+	nav, nav a {
+		font-size: 1.2rem;	
+	}	
+	
+	.about { 
+		padding: 40px 10px;
+	}
+	
+	.profileImage {
+		text-align: center;
```
Comment (charliecog) [inline style.css]: dont need this as its already been defined in the 1000px media query

File: style.css [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+	margin-right: 20px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	padding: 10px 30px 30px 30px;
+}
+
+.projectBox h3 {
+	width: 100%;
+	text-align: center;
```
Comment (charliecog) [inline style.css:125]: the width and text-align can both be removed to achieve the same effect

File: style.css [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+	color: #002d5d;
+	font-size: 1.4rem;
+}
+
+.projectLinks a:hover {
+	color: #2eccd3;
+}
+
+.contact {
+	width: 100%;
```
Comment (charliecog) [inline style.css]: this isnt doing anything as divs are block level elements by default so will take up 100% width without having to specify it

File: style.css [hunk 9]
```diff
@@ -0,0 +1,391 @@
...
+
+.contact p {
+	font-size: 2rem;
+	padding: 20px 140px 30px 140px;
+	margin: 0;
+	
+}
+
+footer {
+	width: 100%;
```
Comment (charliecog) [inline style.css]: this isnt doing anything as footers are block level elements by default so will take up 100% width without having to specify it

**Review by charliecog — COMMENTED** [review]
Looks great, almost there.

General comments:
- remove any files not being used eg: francesco-ungaro-Vjiuta12kBY-unsplash.jpg
- it would be a good idea to rename your headshot photo image file to a name that is more semantic eg: headshot.jpeg
-

File: style.css (hunk 2 above)
Comment (grace-p-03) [reply style.css:68]: I kept these in to avoid my project box headers misaligning

File: style.css (hunk 7 above)
Comment (grace-p-03) [reply style.css:125]: I kept these in to avoid my project box headers misaligning

File: style.css (hunk 2 above)
Comment (grace-p-03) [reply style.css]: I kept text-align: center; on header .intro to keep my title centred with the line below it.

**Review by charliecog — APPROVED** [review]

**Pull request merged**